src/TestSuite/Fixture/ChecksumTestFixture.php

```diff
@@ -21,27 +23,27 @@ class ChecksumTestFixture extends TestFixture
      *
      * @var array<string, string>
      */
-    protected static $_tableHashes = [];
+    protected static array $tableHashes = [];
```

Comment:
The removal of underscore from the properties and methods is unnecessary noise.

Reply:
restored

Reply:
@arusinowski the following methods are still changed:

- `_getTableKey()`
- `_tableUnmodified()`
- `_hash()`